src/jtag/drivers/ftdi.c

```diff
@@ -83,6 +84,18 @@
 /* FTDI access library includes */
 #include "mpsse.h"
 
+#define DEBUG_IO(expr...) LOG_DEBUG(expr)
```

Comment:
DEBUG_JTAG_IO might be more suitable than LOG_DEBUG. Its output is only displayed with -d4, and only compiled in if the OpenOCD build is configured with --enable-verbose-jtag-io.